packages/common-grpc/test/service.js

```diff
@@ -273,33 +274,40 @@ describe('GrpcService', function() {
     });
 
     it('should default grpcMetadata to empty metadata', function() {
-      var fakeGrpcMetadata = {};
+      var fakeGrpcMetadata = {
+        'x-goog-api-client': [
+          'gl-node/' + process.versions.node,
+          'gccl/' + CONFIG.packageJson.version,
+          'grpc/' + grpcVersion].join(' ')
```

Comment:
`].join(' ')` should be on the next line, matching the indentation from where the array was opened.